server/lib/redis.ts

```diff
@@ -4,41 +4,63 @@ import { createClient, RedisClientType } from 'redis';
 
 import logger from './logger';
 
-let redisClient;
-
-export async function createRedisClient(): Promise<RedisClientType> {
-  if (!redisClient) {
-    const url = get(config, 'redis.serverUrl');
-    if (!url) {
-      return;
-    }
-
-    const redisOptions = { url };
-    if (redisOptions.url.includes('rediss://')) {
-      redisOptions['socket'] = { tls: true, rejectUnauthorized: false };
-    }
-
-    redisClient = createClient(redisOptions);
-    try {
-      redisClient.on('error', err => logger.error(`Redis error`, err));
-      redisClient.on('reconnecting', () => logger.info(`Redis reconnecting`));
-      redisClient.on('connect', () => logger.info(`Redis connected`));
-      redisClient.on('ready', () => logger.info(`Redis ready`));
-      redisClient.on('end', () => logger.info(`Redis connection closed`));
-
-      await redisClient.connect();
-    } catch (err) {
-      logger.error(`Redis connection error`, err);
-      redisClient = null;
-    }
+export enum RedisInstanceType {
+  DEFAULT = 'DEFAULT',
+  TIMELINE = 'TIMELINE',
+}
+
+const RedisInstanceKey = {
+  [RedisInstanceType.DEFAULT]: 'redis.serverUrl',
+  [RedisInstanceType.TIMELINE]: 'redis.serverUrlTimeline',
+};
+
+const redisClients: Record<string, RedisClientType> = {};
```

Comment:
```suggestion
const redisInstances: Record<string, RedisClientType> = {};
```

Perhaps, for consistency.